src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1897,32 +2308,431 @@ namespace WaterToAirHeatPumpSimple {
+                        state,
+                        state.dataOutRptPredefined->pdchWAHPRatedCapAtRatedCdts,
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).Name,
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).RatedCapCoolAtRatedCdts);
+                    // update Cooling Coils output reports
+                    OutputReportPredefined::PreDefTableEntry(
+                        state,
+                        state.dataOutRptPredefined->pdchCoolCoilTotCap,
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).Name,
+                        RatedCapCoolTotalDes);
+                    BaseSizer::reportSizerOutput(
+                        state,
+                        "COIL:" +
+                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).WatertoAirHPType +
+                            ":WATERTOAIRHEATPUMP:EQUATIONFIT",
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).Name,
+                        "Design Size Rated Total Cooling Capacity [W]",
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).RatedCapCoolTotal);
+                    BaseSizer::reportSizerOutput(
+                        state,
+                        "COIL:" +
+                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).WatertoAirHPType +
+                            ":WATERTOAIRHEATPUMP:EQUATIONFIT",
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).Name,
+                        "Design Size Rated Sensible Cooling Capacity [W]",
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).RatedCapCoolSens);
+                } else if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).WAHPPlantType ==
+                           DataPlant::PlantEquipmentType::CoilWAHPCoolingEquationFit) { // case 2: companion coil is of EquationFit type but is
+                                                                                        // not autosized
+                    RatedCapHeatDes =
+                        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum).RatedCapCoolTotal *
+                        simpleWatertoAirHP.RatioRatedHeatRatedTotCoolCap;
+                } else { // case 3: companion type is different than EquationFit
+                    RatedCapHeatDes = state.dataSize->DXCoolCap;
```

Comment:
Coil and companion coil adjustments after heating design day calculations. Similarly to the cooling coil adjustments, three cases have been identified and they are all documented in the comments.